src/client/components/HealthCareApp.jsx

```diff
@@ -148,10 +148,11 @@ class HealthCareApp extends React.Component {
         </div>
         <div className="medium-8 columns">
           <div className="progress-box">
-            <div className="form-panel">
+          {/* TODO: Change action to reflect actual action for form submission. */}
+            <form className="form-panel" autoComplete="on">
```

Comment:
Do we need to specify `autocomplete="on"` for this to work? I think `on` is the default.